tools/operator/launcher/k8s/network.go

```diff
@@ -2,241 +2,522 @@ package k8s
 
 import (
 	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+
+	v12 "k8s.io/api/core/v1"
+	apiResource "k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
 
 	"github.com/hyperledger/fabric-test/tools/operator/launcher/nl"
 	"github.com/hyperledger/fabric-test/tools/operator/logger"
-	"github.com/hyperledger/fabric-test/tools/operator/networkclient"
+
+	"github.com/hyperledger/fabric-test/tools/operator/fabricconfig"
 	"github.com/hyperledger/fabric-test/tools/operator/networkspec"
 	"github.com/hyperledger/fabric-test/tools/operator/paths"
-	"github.com/pkg/errors"
 )
```

Comment:
Please remove the whitespace between imports and sort them with the `goimports` tools

Reply:
fixed